pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -121,17 +123,14 @@ func validatePlacementPolicy(policy *placementv1beta1.PlacementPolicy) error {
 
 func validatePolicyForPickFixedPlacementType(policy *placementv1beta1.PlacementPolicy) error {
 	allErr := make([]error, 0)
-	if len(policy.ClusterNames) == 0 {
-		allErr = append(allErr, fmt.Errorf("cluster names cannot be empty for policy type %s", policy.PlacementType))
-	}
```

Comment:
why remove this? I see the scheduling code that treat this as not picking anything. I think we should still block it.